bin/gftools-qa.py

```diff
@@ -35,244 +33,12 @@
     mkdir,
 )
 from gftools.html import HtmlProof, HtmlDiff
```

Comment:
This import needs moving. Can do this if you'd like?

Reply:
Oops. Yes, thanks!